### PR TITLE
improvement/Connection Identity Key Loss Prevention

### PR DESCRIPTION
**Pull request by alazarevski**
Connection loss prevention from api errors

### CODE REVIEW

File: src/actions/authActions.js [hunk 1]
```diff
@@ -120,12 +119,9 @@ export const loginAction = (pin: string, touchID?: boolean = false, onLoginSucce
...
         await dispatch(signalInitAction({ ...signalCredentials, ...OAuthTokensObject }));
         user = merge({}, user, userInfo);
-        if (generateNewConnKeys) {
-          await dispatch(updateConnectionKeyPairs(wallet.mnemonic, wallet.privateKey, user.walletId));
-        } else {
-          dispatch(updateConnectionsAction(user.walletId));
-        }
         dispatch(saveDbAction('user', { user }, true));
+        await
+        dispatch(updateConnectionKeyPairs(wallet.mnemonic, wallet.privateKey, user.walletId, generateNewConnKeys));
```
Comment (jsidorenko) [inline src/actions/authActions.js]: During the biometrics login, you won't get a mnemonic and private key

Comment (jsidorenko) [reply src/actions/authActions.js]: It's better to wrap this call with `if(generateNewConnKeys)..` or you need to change the method's signature and make the first and second params optional

Comment (alazarevski) [reply src/actions/authActions.js]: It is fine as long as the generation flow is wrapped in a condition with the last bool parameter 'generateNewConnKeys'

Comment (alazarevski) [reply src/actions/authActions.js]: Added changes in f0ac452

**Pull request merged**